test/e2e-tests.sh

```diff
@@ -91,6 +91,7 @@ toggle_feature tag-header-based-routing Disabled
 
 toggle_feature allow-zero-initial-scale true config-autoscaler || fail_test
 go_test_e2e -timeout=2m ./test/e2e/initscale ${TEST_OPTIONS} || failed=1
+go_test_e2e -timeout=2m ./test/e2e/activationscale ${TEST_OPTIONS} || failed=1
```

Comment:
Is this behind a feature flag? If not we could just have it under `test/e2e`

Reply:
Nah, this was just to use `initial-scale: 0`... I can move it back up

Reply:
that'll also remove the need to expose the autoscale functions...